public/javascripts/Gallery/src/cards/CardContainer.js

```diff
@@ -92,6 +92,7 @@ function CardContainer(uiCardContainer, initialFilters) {
         sg.ui.cardContainer.holder.on('click', '.static-gallery-image, .additional-count',  (event) => {
             $('.gallery-modal').attr('style', 'display: flex');
             $('.grid-container').css("grid-template-columns", "1fr 5fr");
+            modal.open = true;
```

Comment:
Is there a reason that this is being done here instead of just in the `openModal()` function?

Reply:
Sorry about that! For some reason I didn't see that function.